### ISSUE TITLE
[Bug]: NPE in showMessageView OS 5.0.1

### ISSUE THREAD

**Issue by jdonlan** [OneSignal/OneSignal-Android-SDK#1834]
### What happened?

I am able to init and connect to OS with the new User paradigm. After dismissing an in-app notification, I get a crash after 3-5 seconds with the following stacktrace:

### Steps to reproduce?

```Markdown
1. Update to OS 5.0.1
2. Init and Assign User
3. Add data tags to user
4. Display In-App Message
5. Dismiss message using in-message CTA.
6. Wait
```

### What did you expect to happen?

I expected nothing to happen. 

### OneSignal Android SDK version

5.0.1

### Android version

13

### Specific Android models

```Markdown
Samsung Galaxy S21U
```

### Relevant log output

```Shell
java.lang.NullPointerException
                                                                                                    	at com.onesignal.inAppMessages.internal.display.impl.WebViewManager.showMessageView(WebViewManager.kt:288)
                                                                                                    	at com.onesignal.inAppMessages.internal.display.impl.WebViewManager.access$showMessageView(WebViewManager.kt:40)
                                                                                                    	at com.onesignal.inAppMessages.internal.display.impl.WebViewManager$showMessageView$1.invokeSuspend(Unknown Source:15)
                                                                                                    	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
                                                                                                    	at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:270)
                                                                                                    	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
                                                                                                    	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
                                                                                                    	at kotlinx.coroutines.DispatchedTaskKt.resume(DispatchedTask.kt:235)
                                                                                                    	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:168)
                                                                                                    	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:474)
                                                                                                    	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl(CancellableContinuationImpl.kt:508)
                                                                                                    	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$default(CancellableContinuationImpl.kt:497)
                                                                                                    	at kotlinx.coroutines.CancellableContinuationImpl.resumeUndispatched(CancellableContinuationImpl.kt:595)
                                                                                                    	at kotlinx.coroutines.android.HandlerContext$scheduleResumeAfterDelay$$inlined$Runnable$1.run(Runnable.kt:19)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:942)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:226)
                                                                                                    	at android.os.Looper.loop(Looper.java:313)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8757)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
Hello @jdonlan thank you for reaching out! I've followed the steps you provided but have been unable to reproduce this issue. Can you share more about the In-App Message you've set up (any custom triggers you might be adding?) and any other relevant log output preceding this crash? It would also be helpful to know whether you've experienced this on multiple devices and if this is happening every time or only sporadically.

Thanks!

**Comment by jdonlan** [comment]
I think I found the bug. The issue is when an in-app message is set to auto-dismiss after X seconds, but has already been manually dismissed. This is a new problem in OS5+. I can reproduce the issue consistently by creating an in-app message with an auto dismiss and observing the following behaviors:

1. Allow the timer to count down and auto-dismiss - works as expected. 
2. Manually dismiss the in-app-message and wait for timer to complete and produce an NPE crash.

**Comment by emawby** [comment]
Thank you for the investigation that is very helpful! We will work on a fix immediately

**Comment by jdonlan** [comment]
This issue is blocking and causing us to delay our migration from OS4 to OS5. Just wondering if there is any sort of ETA on it based on the "fix immediately" comment above.

**Comment by jennantilla** [comment]
@jdonlan thank you for your patience! This fix has been included in our latest release, [5.0.2](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/5.0.2). Please let us know if you have any additional questions or concerns!
